charts/astronomer/templates/houston/houston-automatic-chart-upgrade.yaml

```
@@ -0,0 +1,36 @@
+#######################################
+## Airflow Post Deployment Upgrade Hook
+#######################################
+{{- if .Values.houston.autoUpgradeDeployments.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-deployment-post-upgrade-hook"
+  labels:
+    tier: astronomer
+    component: houston
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-upgrade
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: post-upgrade-job
+          image: {{ template "houston.image" . }}
+          imagePullPolicy: {{ .Values.images.houston.pullPolicy }}
+          command: ["npm", "run", "deployment-upgrade"]
```

```suggestion
          command: ["npm", "run", "upgrade-deployments"]
```

@andriisoldatenko I modified the command names and directory structure a bit - https://github.com/astronomer/houston-api/blob/master/package.json#L18-L19.